src/Moryx.ControlSystem/VisualInstructions/ActiveInstruction.cs

```diff
@@ -31,8 +32,8 @@ public class ActiveInstruction
         public object Inputs { get; set; }
 
         /// <summary>
-        /// Results of the instruction
+        /// Possible results of the instruction
         /// </summary>
-        public IReadOnlyList<string> PossibleResults { get; set; } = new string[0];
+        public IReadOnlyList<InstructionResult> Results { get; set; } = Array.Empty<InstructionResult>();
```

Comment:
For the `EnumInstructionResult` you left the old method `PossibleInstructionResults` in there with an obsolete attribute. Here I would suggest either doing it the same way, or specifically adding the Renaming of this property to the migration guide. otherwise people will just wonder why the property isn't there anymore and hope, that the new property does the same thing after they found it.

Reply:
That is technically not a renaming. It removes the obsolote from `dev`, Git just shows the diff weird. In `EnumInstructionResult` I also removed the string version and renamed the method introduced with 8.1, but kept the old-new method.